### PR TITLE
Work

### PR DESCRIPTION
**Pull request by mschwartz**
Added routing.  Tapping on tile buttons now changes screens.

Some cleanup.

### CODE REVIEW

File: src/Tablet/Dashboard/PoolTile.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ const PoolTile = ({ device }) => {
       height={1}
       onClick={() => {
         localStorage.setItem("autelis-radio", "pool");
-        window.location.hash = "poolcontrol";
+        window.location.hash = "autelis";
```
Comment (grgur) [inline src/Tablet/Dashboard/PoolTile.js]: we already have react router as a dependency so we shouldn't have to reinvent this. 
Let's not cross that bridge right now, just wanted to give a heads up.

File: src/Tablet/Dashboard/TheaterTile.js [hunk 2]
```diff
@@ -165,7 +165,7 @@ const TheaterTile = ({ title }) => {
   };
 
   return active ? (
-    <Tile width={2} height={2}>
+    <Tile width={2} height={2} onClick="theater">
```
Comment (grgur) [inline src/Tablet/Dashboard/TheaterTile.js]: how is onClick a string?

File: src/Tablet/Dashboard/WeatherTile.js [hunk 3]
```diff
@@ -39,7 +40,9 @@ const WeatherTile = () => {
             }}
             src={"/img/Weather/icons/black/" + now.icon + ".svg"}
           />
-          <div style={{ display: "inline", paddingTop: 10 }}>{now.current_temperature}&deg;F</div>
+          <div style={{ display: "inline", paddingTop: 10 }}>
```
Comment (grgur) [inline src/Tablet/Dashboard/WeatherTile.js]: We should really move away from styles and use regular css. I really feel good about css modules that work well with CRA. We can talk about that in specific later. No need to refactor right now

File: src/Tablet/MainScreen.js [hunk 4]
```diff
@@ -73,9 +103,9 @@ const MainScreen = () => {
             bg="primary"
             variant="dark"
             onSelect={tab => {
-              localStorage.setItem(LOCALSTORAGE_KEY, tab);
-              setActiveTab(tab);
-              console.log("eventKey");
+              window.location.hash = "#" + tabInfo[tab];
+              //              localStorage.setItem(LOCALSTORAGE_KEY, tab);
```
Comment (grgur) [inline src/Tablet/MainScreen.js]: can we delete the commented out code?

File: src/Tablet/Dashboard/TheaterTile.js (hunk 2 above)
Comment (mschwartz) [reply src/Tablet/Dashboard/TheaterTile.js]: if not a function, it sets window.location.has to "#theater"

File: src/Tablet/Dashboard/PoolTile.js (hunk 1 above)
Comment (mschwartz) [reply src/Tablet/Dashboard/PoolTile.js]: I originally looked at react-router and used it in my original client code.

We don't really have any global state in the app, and I prefer to keep it that way (for now).  By global state, I mean redux or some other scheme to hoist state to parent(s) of a component.

Setting the hash is a simple assignment.  The hash change is honored.

Beyond this triviality, it is important that the localStorage stuff is done at the same time.  When run as a PWA, I don't want to navigate to the Pool Control screen, change to another app, change back to the PWA and have it reload/redirect back to the dashboard.  It remembers where to start/restart from localStorage.  It's particularly nice when I reboot my tablet, which is dedicated to a specific room, and it comes up in the PWA showing the dashboard for that room, as I left off.

File: src/Tablet/MainScreen.js (hunk 4 above)
Comment (mschwartz) [reply src/Tablet/MainScreen.js]: yes

File: src/Tablet/Dashboard/WeatherTile.js (hunk 3 above)
Comment (mschwartz) [reply src/Tablet/Dashboard/WeatherTile.js]: I hate CSS.

There are memes all over the internet about how awful it is.

Abramov really likes css in js, so do I :)

**Comment by mschwartz** [comment]
@grgur

**Pull request merged**